src/main/avro/input/beskjedInput.avsc

```diff
@@ -0,0 +1,48 @@
+{
+    "type": "record",
+    "namespace": "no.nav.brukernotifikasjon.schemas.input",
+    "name": "BeskjedInput",
```

Comment:
Trenger vi å endre navn på eventtypene tro? For dette blir vel det navnet produsentene våre forholder seg til?

Reply:
Var litt usikker. I første omgang tenkte det var mest for å signalisere at her er det noe nytt, men er ikke låst til den navngivingen.

Reply:
Kanskje det beste hadde vært å gi nytt navn til de gamle klassene i stedet? Er litt imot at de har presist samme navn, siden det ofte er frustrerende å jobbe med.

Reply:
Ja, enig i det. Kanskje best å gi til de gamle, ja.